Kinvey/Kinvey/Geolocation.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  Geolocation.swift
+//  Kinvey
+//
+//  Created by Victor Hugo on 2017-02-01.
+//  Copyright © 2017 Kinvey. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+import ObjectMapper
+import CoreLocation
+import MapKit
+
+open class GeoPoint: Object, StaticMappable {
+    
+    dynamic var latitude: CLLocationDegrees = 0.0
+    dynamic var longitude: CLLocationDegrees = 0.0
+    
+    public static func objectForMapping(map: Map) -> BaseMappable? {
+        guard let _: Double = map["latitude"].value(), let _: Double = map["longitude"].value() else {
+            return nil
+        }
+        return GeoPoint()
+    }
+    
+    public convenience init(latitude: Double, longitude: Double) {
```

Comment:
@heyzooi if we have the constructor for `CLLocation`, what is the reason to have this one separately? I prefer we keep one way of doing things when possible.